INA219.cpp

```diff
@@ -149,9 +156,16 @@ void INA219::configure(  t_range range,  t_gain gain,  t_adc  bus_adc,  t_adc sh
 }
 
 #define INA_RESET        0xFFFF    // send to CONFIG_R to reset unit
-void INA219::reset(){
-  write16(CONFIG_R, INA_RESET);
-  _delay_ms(5);
+//RMW Returns false if unable to reset (likely due to device not being functional/installed)
+bool INA219::reset(){
+  uint8_t ret = write16(CONFIG_R, INA_RESET);
+  if (ret) {
+    Serial.print("INA219::reset: write16 error: ");
```

Comment:
No Serial.print in library

Reply:
Understood, but kind of a bummer that the error value doesn't wind up bubbling up to the top as it goes from a real value here to just a success or failure with the bool. Maybe bubbling the error value up would be wiser if you don't want Serial.print in the library.